lib/core/src/Cardano/Byron/Codec/Cbor.hs

```diff
@@ -861,3 +858,10 @@ decodeNestedBytes dec bytes =
             fail "Leftovers when decoding nested bytes"
         _ ->
             fail "Could not decode nested bytes"
+
+deserialise
```

Comment:
What about calling it `deserialiseCbor` to make it the dual of `unsafeDeserialiseCbor`

Reply:
OK done